Userland/Utilities/dd.cpp

```diff
@@ -142,36 +143,38 @@ int main(int argc, char** argv)
     uint8_t* buffer = nullptr;
     ssize_t nread = 0, nwritten = 0;
 
-    for (int a = 1; a < argc; a++) {
-        if (!strcmp(argv[a], "--help")) {
+    for (size_t a = 1; a < arguments.strings.size(); a++) {
+        auto argument = arguments.strings[a];
+
+        if (argument == "--help") {
             out("{}", usage);
             return 0;
-        } else if (!strncmp(argv[a], "if=", 3)) {
-            if (handle_io_file_arguments(input_fd, input_flags, argv[a]) < 0) {
+        } else if (argument.starts_with("if=")) {
+            if (handle_io_file_arguments(input_fd, input_flags, argument.characters_without_null_termination()) < 0) {
                 return 1;
             }
-        } else if (!strncmp(argv[a], "of=", 3)) {
-            if (handle_io_file_arguments(output_fd, output_flags, argv[a]) < 0) {
+        } else if (argument.starts_with("of=")) {
+            if (handle_io_file_arguments(output_fd, output_flags, argument.characters_without_null_termination()) < 0) {
                 return 1;
             }
-        } else if (!strncmp(argv[a], "bs=", 3)) {
-            if (handle_size_arguments(block_size, argv[a]) < 0) {
+        } else if (argument.starts_with("bs=")) {
+            if (handle_size_arguments(block_size, argument.characters_without_null_termination()) < 0) {
                 return 1;
             }
-        } else if (!strncmp(argv[a], "count=", 6)) {
-            if (handle_size_arguments(count, argv[a]) < 0) {
+        } else if (argument.starts_with("count=")) {
+            if (handle_size_arguments(count, argument.characters_without_null_termination()) < 0) {
                 return 1;
             }
-        } else if (!strncmp(argv[a], "seek=", 5)) {
-            if (handle_size_arguments(seek, argv[a]) < 0) {
+        } else if (argument.starts_with("seek=")) {
+            if (handle_size_arguments(seek, argument.characters_without_null_termination()) < 0) {
                 return 1;
             }
-        } else if (!strncmp(argv[a], "skip=", 5)) {
-            if (handle_size_arguments(skip, argv[a]) < 0) {
+        } else if (argument.starts_with("skip=")) {
+            if (handle_size_arguments(skip, argument.characters_without_null_termination()) < 0) {
                 return 1;
             }
-        } else if (!strncmp(argv[a], "status=", 7)) {
-            if (handle_status_arguments(status, argv[a]) < 0) {
+        } else if (argument.starts_with("status=")) {
+            if (handle_status_arguments(status, argument.characters_without_null_termination()) < 0) {
```

Comment:
Can we fix all of these handle functions to just take StringView's instead?

Reply:
Yes, done like suggested! :)